packages/create-fuels/test/utils/generateArgs.ts

```diff
@@ -1,18 +1,26 @@
-export const generateArgs = (projectName?: string, packageManager: string = 'pnpm'): string[] => {
+export const generateArgs = (
+  projectName?: string,
+  packageManager: string = 'pnpm',
+  template?: string
+): string[] => {
```

Comment:
May be worth making this a single parameter with an object, just to avoid confusion in tests

Reply:
➕ for named params.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2851/commits/3eac3a1fce009ce7564eb7c0e5ce789aa5447b7b